requirements.txt

```diff
@@ -56,7 +56,8 @@ kubernetes~=11.0
 #  separating the SDK and API code) (referring to humanfriendly and fastapi)
 humanfriendly~=8.2
 fastapi~=0.62.0
-fsspec~=0.9.0
+# fsspec versioning changed to year.month.patch after 0.9.0
```

Comment:
Explain why you're upbound-ing to 2021.8.1 (I guess the explanation is that this is just what we tested and since it's not semver we can't assume backwards compatibility)

Reply:
Yes, that's the reason. Wouldn't make sense to use `~=` or an equivalent.